consumer/dragonboard/qualcomm-robotics-rb1/README.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Using the Qualcomm Robotics RB1 Development kit
+permalink: /documentation/consumer/dragonboard/qualcomm-robotics-rb1/
+---
+# Using the Qualcomm® Robotics RB1 Development Kit
+
+## Documentation
+
+- [Quick Start Guide](https://developer.qualcomm.com/hardware/qualcomm-robotics-rb1-rb2-kits/quick-start-guides)
+   - Learn about your development board, how to prepare and set up for basic use
+- [Hardware Documentation](https://developer.qualcomm.com/hardware/qualcomm-robotics-rb1-rb2-kits/hardware-reference-guide)
+   - Explore what makes your development board unique, technical specifications, schematics, hardware notes and more...
+
+## Software
+
+### With upstream Linux versions by Linaro
+
+A comprehensive guide to using the [Qualcomm® Robotics RB1 Development Kit](https://www.qualcomm.com/products/internet-of-things/industrial/industrial-automation/qualcomm-robotics-rb1-platform) . This guide is written by the [96Boards](https://www.96boards.org) team at [Linaro](http://www.linaro.org) with community contributions and links to third-party content.
+
+- [Downloads](downloads/)
+   - Already familiar with your 96Boards? Skip the board bring up sections and go straight to your download!
+- [Installation](installation/)
+   - Choose and install an operating system on your development kit
+
+### With downstream version from Thundercomm
+
+Developers can use the downstream version from Thundercomm to get started. Software and tools to download, build and flash the boot images and rootfs images can be found on Thundercomm's website.
+
+- [Downloads from Thundercomm](https://www.thundercomm.com/product/qualcomm-robotics-rb1-platform/)
```

Comment:
Don't we have the link to the SDK?

Reply:
@Mani-Sadhasivam 

There is a direct link to an SDK manager archive file (https://thundercomm.s3.ap-northeast-1.amazonaws.com/uploads/web/common/TC-sdkmanager-4.0.0.zip), but I am reluctant to add this link to a file hosted on AWS on the 96boards pages.

Also to my knowledge it might be preferred to download boards' STD archive with image files, what's your opinion?

Reply:
You should point to the actual archive that the customer can download and use. Directing them to the board page doesn't help.

Reply:
I dont think TC releases them w.o registering so not sure we can do much about it

Reply:
The page should be `https://www.thundercomm.com/product/qualcomm-robotics-rb1-platform/#sdk-manager`